src/sbvr-api/sbvr-utils.ts

```diff
@@ -1205,17 +1287,22 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 			await runHooks('PREPARSE', reqHooks, { req, tx: req.tx });
 			let requests: uriParser.UnparsedRequest[];
 			// Check if it is a single request or a batch
-			if (req.batch != null && req.batch.length > 0) {
-				requests = req.batch;
+			if (req.url === `/${vocabulary}/$batch`) {
+				requests = req.batch!;
```

Comment:
I expect `req.batch` can be removed entirely now that the batch is just json in `req.body`

Reply:
Pushed in this commit: https://github.com/balena-io/pinejs/pull/675/commits/e01ace83e102b68fc0bef5911821a01ef8965122

Is this what you had in mind?